BaseTools/Scripts/PatchCheck.py

```diff
@@ -209,6 +210,14 @@ def find_signatures(self, sig):
 
         return sigs
 
+    def check_subject(self, updated_packages):
+        if updated_packages:
+            for package in updated_packages:
+                current_package_re = r".*" + re.escape(package) + r".*:.*"
+                if not re.search(current_package_re, self.subject):
+                    self.error("Subject line does not follow required format.")

```

Comment:
Thanks for hacking this up. 

Might we be a bit more helpful here in describing how the subject line does not follow the required format?